src/css/SpellBox.css

```diff
@@ -0,0 +1,19 @@
+.spellBox {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    margin: 10%;
+    padding: 5%;
+    border: 5px solid black;
+}
+
+.singleSpell {
+    display: flex;
+    justify-content: space-between;
+    font-size: large;
+    width: 80%;
+    padding: 5%;
+    margin: 5%;
+    background-color: rgb(239, 204, 253);
```

Comment:
It cracks me up that our layout is purple & pinkish now 😂

Reply:
We NEED to fix that!

Reply:
Lol yeah, definitely not what we started with. It has sort of grown on me.... I would still like to take a stab at having an actual cohesive color palette and consistency in layout.